pinconfigs/sam-c21.yml

```diff
@@ -0,0 +1,736 @@
+# Copyright (c) 2022 Kamil Serwus
+# SPDX-License-Identifier: Apache-2.0
+#
+# Sources:
+# - SAM_C20_C21_Family_Data_Ssheet_DS60001479D (Revision D - 01/2020)
```

Comment:
This file name should be sam-c2x.yml
Since you are introducing the pin definitions you need provide for both variations. In this case, C21 is the full set and you just need add exclusions for C20. You can check sam-d2x-da1-abcd.yml as example.

Please, fix the name` ..._Ssheet_...`

Reply:
I will try :)